apps/web/src/Pages/Airdrop/Table/Table.tsx

```
@@ -162,268 +177,302 @@ const AirdropTable: React.FC<LockupProps> = ({
+                  </TableCell>
+                  <TableCell
+                    sx={{
+                      color:
+                        rate((selection || contract).global_period) > 0
+                          ? "green"
+                          : "inherit",
+                      fontWeight: 900,
+                      textAlign: "right",
+                    }}
+                  >
+                    {!!selection ? (
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)